lnrpc/routerrpc/router_server_test.go

```diff
@@ -387,7 +388,13 @@ func TestIsLsp(t *testing.T) {
 
 	for _, tc := range lspTestCases {
 		t.Run(tc.name, func(t *testing.T) {
-			require.Equal(t, tc.isLsp, isLSP(tc.routeHints))
+			require.Equal(t, tc.isLsp, isLSP(tc.routeHints, &RouterBackend{
```

Comment:
I think it makes sense to add a unit test for this behaviour.

Reply:
I've added test case for this new behavior.